src/receivers/AwsLambdaReceiver.ts

```diff
@@ -243,4 +247,10 @@ export default class AwsLambdaReceiver implements Receiver {
 
     return true;
   }
+
+  // eslint-disable-next-line class-methods-use-this
+  private getHeaderValue(headers: Record<string, any>, key: string): string | undefined {
+    const caseInsensitivKey = Object.keys(headers).find((it) => key.toLowerCase() === it.toLowerCase());
```

Comment:
@misscoded Thanks, good catch 👍 
@TheManWhoStaresAtCode Can you merge this suggestion or push a new commit fixing this?
```suggestion
    const caseInsensitiveKey = Object.keys(headers).find((it) => key.toLowerCase() === it.toLowerCase());
```